PocketKit/Sources/PocketKit/MyList/ItemsList/ItemsListViewController.swift

```diff
@@ -183,11 +183,11 @@ class ItemsListViewController<ViewModel: ItemsListViewModel>: UIViewController,
             }
         }
 
-        model.events.sink { [weak self] event in
+        model.events.receive(on: DispatchQueue.main).sink { [weak self] event in
```

Comment:
Ensure the view controller always handles save events on the main thread. (all we do is update ui here)